### PR TITLE
Auth and chart

### PR DESCRIPTION
**Pull request by AhmadMaree**
Add use context for both Pages

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1022/112213.383:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (HinnHabib) [inline debug.log]: This should be moved to gitignore 
No need to push

File: src/container/Authenticate/Auth.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    email : Yup.string("Enter your Email")
+                            .email("Enter Vaild Email")
+                            .required("Email is Requird"),
+                    password: Yup.string("")
+                            .min(8, "Password must contain at least 8 characters")
+                            .required("Enter your password"),
+                })}
+            >
+                            {props => <Form {...props}/>}
+            </Formik>
```
Comment (HinnHabib) [inline src/container/Authenticate/Auth.js]: Lets use formik hooks

File: src/container/Todo/Todo.js [hunk 3]
```diff
@@ -67,22 +93,33 @@ const Todo = props =>  {
         ...item,
         checked : !item.checked
       }
-          axios.put(`/ListTodo/${item.id}.json`,updateChecked)
+          axios.put(`/ListTodo/${item.id}.json?auth=`+stateAuth.idToken,updateChecked)
```
Comment (HinnHabib) [inline src/container/Todo/Todo.js]: Can we apply the concept of interceptor for the authentication

Comment (HinnHabib) [reply src/container/Todo/Todo.js]: Will go over tomorrow

**Review by HinnHabib — CHANGES_REQUESTED** [review]

File: src/container/Authenticate/Auth.js (hunk 2 above)
Comment (AhmadMaree) [reply src/container/Authenticate/Auth.js]: Done

**Pull request closed without merging**